tcf_website/views/review.py

```diff
@@ -3,9 +3,13 @@
 
 from django import forms
 from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin # For class-based views
+
```

Comment:
An unnecessary line?

Reply:
tryna group by category for readability

maybe this isn't a good category tho

Reply:
I believe this file already follows the standard in PEP8:
> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Reordered them alphabetically